validator/evaluation/docker_evaluation.py

```diff
@@ -63,12 +63,12 @@ async def run_evaluation_docker(
         )
         # NOTE: replace with asyncio.to_thread
 
-        log_thread = threading.Thread(target=stream_logs, args=(container,))
-        log_thread.start()
+#        log_thread = threading.Thread(target=stream_logs, args=(container,))
```

Comment:
intended to stay commented out? If so, can we delete? If not, perhaps a # TODO: or # NOTE: to remove comment

Reply:
Ah this is me playin with things to avoid getting stuck with loading a model